unit-tests-java8/src/test/java/org/eclipse/collections/test/set/SetIterableTestCase.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2021 Goldman Sachs.
```

Comment:
Why is this only a copyright update in this file while all other files do not have copyright updated?

Reply:
found two files like this and reverted them